cloudofficeprint/src/main/java/com/cloudofficeprint/RenderElements/Cells/CellStyleDocxPpt.java

```diff
@@ -38,33 +67,526 @@ public String getWidth() {
      *                        table)). Giving a width of 0 will remove the whole
      *                        column.
+     * @param preserveTotalWidthOfTable Keeps table width constant by redistributing removed column's width to others..
+     * @param border The border style applied to all edges (top, bottom, left, right) of the table. Optional.
+     * @param borderTop The border style applied to the top edge of the table. Optional.
+     * @param borderBottom The border style applied to the bottom edge of the table. Optional.
+     * @param borderLeft The border style applied to the left edge of the table. Optional.
+     * @param borderRight The border style applied to the right edge of the table. Optional.
+     * @param borderDiagonalDown Applies the specified border style to the diagonal line going from the top-left to the bottom-right corner. Optional.
+     * @param borderDiagonalUp  Applies the specified border style to the diagonal line going from the bottom-left to the top-right corner. Optional.
+     * @param borderColor The color of the borders (top, bottom, left, right). Optional.
+     * @param borderTopColor  The color of the top border. Optional.
+     * @param borderBottomColor  The color of the bottom border. Optional.
+     * @param borderLeftColor  The color of the left border. Optional.
+     * @param borderRightColor  The color of the right border. Optional.
+     * @param borderDiagonalUpColor  The color of the diagonal up border. Optional.
+     * @param borderDiagonalDownColor The color of the diagonal down border. Optional.
+     * @param borderSize The width of the borders (top, bottom, left, right) in points. Optional.
+     * @param borderTopSize  The width of the top border in points. Optional.
+     * @param borderBottomSize  The width of the bottom border in points. Optional.
+     * @param borderLeftSize  The width of the left border in points. Optional.
+     * @param borderRightSize  The width of the right border in points. Optional.
+     * @param borderDiagonalUpSize  The width of the diagonal up border in points. Optional.
+     * @param borderDiagonalDownSize  The width of the diagonal down border in points. Optional.
+     * @param borderSpace The spacing between the content and borders in points. Optional.
+     * @param borderTopSpace  The spacing between the content and the top border. Optional.
+     * @param borderBottomSpace The spacing between the content and the bottom border. Optional.
+     * @param borderLeftSpace The spacing between the content and the left border. Optional.
+     * @param borderRightSpace The spacing between the content and the right border in points Optional.
+     * @param borderDiagonalUpSpace The spacing between the content and the diagonal up border in points. Optional.
+     * @param borderDiagonalDownSpace The spacing between the content and the diagonal down border in points. Optional.
      */
-    public CellStyleDocxPpt(String backgroundColor, String width) {
```

Comment:
signature for the constructor cannot change for backwards compatibility, create another constructor with the different signature.